doc/developer_guide/developer-guide.md

```diff
@@ -0,0 +1,84 @@
+# Notebook Connector Developer Guide
+
+## How to Release?
+
+Creating a new release of the NC requires 4 steps
```

Comment:
Only 3 steps are listed

Reply:
Yes :rofl: 
Initially it was 4 or 5, I then could reduce to 3 and now it's 4 again. 
Please see latest push.